conf/nonclangable.conf

```diff
@@ -201,6 +201,9 @@ TOOLCHAIN:pn-gcc-for-nvcc-runtime:tegra = "gcc"
 #|       | ^
 TOOLCHAIN:pn-optee-os:imx-nxp-bsp = "gcc"
 
+# Does not compile with clang 18
+TOOLCHAIN:pn-bpftool = "gcc"
```

Comment:
Its an independent tool and infact should work well with clang, please fix the code.